tdp/cli/commands/init.py

```diff
@@ -23,13 +23,13 @@
 
 
 @click.command()
-@overrides_option
+@conf_option
 @collections_option
 @database_dsn_option
 @validate_option
 @vars_option(exists=False)
 def init(
-    overrides: tuple[Path],
+    user_vars: tuple[Path],
```

Comment:
Conf option map to `user_vars`, why ??? It is confusing.

Reply:
`--conf` option naming is only scoped to the `cli` package. Once in the `core` package, I used `user_vars` everywhere which goes very well with `vars`.

IMO it would be confusing to call it `conf` in the `core` package. However, I can call use `conf` for Click parameter injection if this is what you find confusing.

Reply:
I think it should be call `conf` everywhere to avoid confusion like before was called `overrides` and the word `override` is in the core.

Reply:
`conf` is probably the most confusing naming we can use to designate user variables in `core`. Why not using something specific to designate what it really is?